policy/modules/services/dbus.te

```diff
@@ -205,7 +215,7 @@ ifdef(`init_systemd', `
 	')
 ')
 
-tunable_policy(`dbus_pass_tuntap_fd',`
+tunable_policy(`dbus_pass_tuntap_fd && dbus_can_network',`
```

Comment:
Even if dbus has no networking, it may still need to pass a tun device, and this access is required.

Reply:
Amended